src/App.tsx

```diff
@@ -1,17 +1,20 @@
 import { AppBar } from "./components";
 import LeftBar from "./components/LeftBar/LeftBar";
```

Comment:
Make imports consistency. It should look like: 
`import { AppBar, LeftBar, MainContent } from "./components";`